roles/internal/Islandora-Devops.alpaca/defaults/main.yml

```diff
@@ -3,80 +3,71 @@
 alpaca_from_source: no
 alpaca_version: main
 alpaca_clone_directory: /opt/alpaca
+alpaca_jar_path: "{{ alpaca_clone_directory }}/islandora-alpaca-app/build/libs/islandora-alpaca-app-2.0.0-all.jar"
 
-alpaca_karaf_repos:
-  - mvn:org.apache.camel.karaf/apache-camel/2.20.4/xml/features
-  - mvn:org.apache.activemq/activemq-karaf/5.15.0/xml/features
-  - mvn:ca.islandora.alpaca/islandora-karaf/1.0.1/xml/features
+alpaca_user: tomcat
+alpaca_group: tomcat
```

Comment:
Should this create its own user/group by default? Just yeah... that coupling thing again.

Reply:
Changed these to `vagrant`

Reply:
Apologies if this is not an issue, but can we make sure you aren't hardcoding 'vagrant' in there since we aren't always deploying to vagrant.

{{ ansible_os_user }} is the most current way to reference the user that becomes 'vagrant' on a vagrant box.

The Drupal role gives a more robust expression:

```yaml
 "{{ ansible_ssh_user | default(ansible_env.SUDO_USER, true) | default(ansible_env.USER, true) | default(ansible_user_id) }}"
```

Reply:
I'm still using a role variable, sorry I should have been clearer. I just changed the default from `tomcat` to `vagrant`, so you can still change this to whatever you want by overridding it in your inventory.